fling/component/group/fedcac_group.py

```diff
@@ -0,0 +1,130 @@
+import time
+import copy
+
+import torch
+
+from fling.utils import get_params_number
+from fling.utils.compress_utils import *
+from fling.utils.registry_utils import GROUP_REGISTRY
+from fling.utils import Logger
+from fling.component.client import ClientTemplate
+from fling.component.group import ParameterServerGroup
+
+@GROUP_REGISTRY.register('fedcac_group')
+class FedCACServerGroup(ParameterServerGroup):
+    r"""
+        Overview:
+            Implementation of the group in FedCAC.
+    """
+    def __init__(self, args: dict, logger: Logger):
+        super(FedCACServerGroup, self).__init__(args, logger)
+        self.epoch = -1  # To be consistent with the existing pipeline interface. group maintains a epoch counter itself.
+
+    def sync(self) -> None:
+        r"""
+            Overview:
+                    Perform the initialization steps in FedCAC (Eq.(11) in the paper)
+        """
+        if self.epoch == -1:
+            super().sync()  # Called during system initialization
```

Comment:
Is it necessary? Maybe it is executed in ``super.initialize()``

Reply:
Yes, it is necessary. Because I override the sync(). In the system initialization, the system will call sync(). At this point, the super().sync() should be invoked to synchronize all local models.